runtime/service.go

```diff
@@ -285,6 +286,7 @@ func (s *service) StartShim(shimCtx context.Context, containerID, containerdBina
 	bundleDir := bundle.Dir(cwd)
 
 	var exitAfterAllTasksGone bool
+	containerCount := 0
```

Comment:
Can you add a comment here or around line 326 explaining that a zero-value here is ignored and why it's ignored?

Reply:
Done.